hw/ip/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson

```diff
@@ -39,6 +39,16 @@
 
   en_build_modes: ["{tool}_crypto_dpi_prince_build_opts"]
 
+  build_modes: [
+    // Sim mode that enables build randomization. See the `build_seed` mode
+    // defined in `hw/dv/tools/dvsim/common_modes.hjson` for more details.
+    {
+      name: build_seed
+      pre_build_cmds: ["cd /${proj_root} && ./util/design/gen-otp-mmap.py --seed ${seed}"]
```

Comment:
I believe you mentioned this in the previous PR, shall we also relax the `$` sign here?

Reply:
Please correct me if I am wrong - I believe this pre_build_cmd will also apply to private CI? Will that be okay? I personally think it is fine but just want to point out, and maybe good to document it here: https://docs.opentitan.org/doc/getting_started/setup_dv/#reproduce-a-dv-failure-in-ci?

Reply:
It won't, unless you pass `--build-seed` on the command line explciitly.

Reply:
Oh! that is right. So when running regression, we just need to add --build-seed to the script! Got it thanks!